src/obstacle_detector.cpp

```diff
@@ -158,14 +165,70 @@ void ObstacleDetector::sensor_callback(const sensor_msgs::Range::ConstPtr &msg)
     else {
         RangeSensor& sensor = sensors[frame];
```

Comment:
Another minor comment, not necessarily part of this PR:

We are effectively running find twice because we are not using the value of the iterator it here.

I think this line could become `RangeSensor& sensor = *it;` which means we are only doing the map lookup once.

Reply:
👍